ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java

```diff
@@ -822,4 +833,51 @@ private String buildCorsRegexFromConfig() {
       return stringJoiner.toString();
     }
   }
+
+  static class JsonResponseStreamer extends OutputStream {
```

Comment:
It might be worth separating this inner class into it's own file, as JsonRpcHttpService is already over 800 lines.

Reply:
done